### PR TITLE
fix: make sure that env vars always exist

### PR DESCRIPTION
**Pull request by odinn1984**
- [ ] Ready for review
- [ ] Follows [CONTRIBUTING](https://github.com/snyk/broker/blob/master/.github/CONTRIBUTING.md) rules

#### What does this PR do?
This fixes a problem where the broker is working correctly but system check does not work because the url in the env vars isn't defined.

### CODE REVIEW

File: dockerfiles/bitbucket-server/Dockerfile [hunk 1]
```diff
@@ -33,39 +33,17 @@ ENV BITBUCKET_PASSWORD <password>
 ENV BITBUCKET your.bitbucket.server.hostname
 
 # The Bitbucket server API URL
-ENV BITBUCKET_API $BITBUCKET/rest/api/1.0
+# ENV BITBUCKET_API $BITBUCKET/rest/api/1.0
```
Comment (adrukh) [inline dockerfiles/bitbucket-server/Dockerfile]: `BITBUCKET` and `BITBUCKET_API` should either both be defined here or commented out.

File: dockerfiles/gitlab/Dockerfile [hunk 2]
```diff
@@ -21,43 +21,23 @@ RUN broker init gitlab
 ######################################
 
 # Your unique broker identifier
-ENV BROKER_TOKEN <broker-token>
+# ENV BROKER_TOKEN <broker-token>
```
Comment (adrukh) [inline dockerfiles/gitlab/Dockerfile]: Leave uncommented?

File: dockerfiles/gitlab/Dockerfile [hunk 3]
```diff
@@ -21,43 +21,23 @@ RUN broker init gitlab
...
 
 # your personal token to your Gitlab server account
-ENV GITLAB_TOKEN <gitlab-token>
+# ENV GITLAB_TOKEN <gitlab-token>
```
Comment (adrukh) [inline dockerfiles/gitlab/Dockerfile]: Leave uncommented?

File: dockerfiles/gitlab/Dockerfile [hunk 4]
```diff
@@ -21,43 +21,23 @@ RUN broker init gitlab
...
 
 # The Gitlab server API URL
-ENV GITLAB your.gitlab.server.hostname
+# ENV GITLAB your.gitlab.server.hostname
```
Comment (adrukh) [inline dockerfiles/gitlab/Dockerfile]: Leave uncommented?

File: client-templates/jira/.env.sample [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
 # your personal username to your Jira Server account
-JIRA_USERNAME=
+JIRA_USERNAME=<jira-username>
 
 # your personal password or API token to your Jira Server account
-JIRA_PASSWORD=
+JIRA_PASSWORD=<jira-password>
 
 # your Jira Server hostname, i.e. jira.yourdomain.com
 JIRA_HOSTNAME=
```
Comment (adrukh) [inline client-templates/jira/.env.sample]: Add a dummy hostname

File: dockerfiles/bitbucket-server/Dockerfile [hunk 6]
```diff
@@ -29,43 +29,15 @@ ENV BITBUCKET_USERNAME <username>
 # Your personal password to your bitbucket server account
 ENV BITBUCKET_PASSWORD <password>
 
-# Your Bitbucket Server host, excluding scheme
-ENV BITBUCKET your.bitbucket.server.hostname
-
-# The Bitbucket server API URL
-ENV BITBUCKET_API $BITBUCKET/rest/api/1.0
-
```
[the remark this answers is not in the document]
Comment (odinn1984) [reply dockerfiles/bitbucket-server/Dockerfile]: but i thought that we want to remove thees kind of things? i moved it to the env file

**Pull request merged**